lib/wallaroo_labs/messages/external_messages.pony

```diff
@@ -146,6 +154,66 @@ primitive ExternalMsgEncoder
     let sis = SourceIdsQueryEncoder.response(source_ids)
     _encode(_SourceIdsQueryResponse(), sis, wb)
 
+  fun state_entity_query(wb: Writer = Writer): Array[ByteSeq] val =>
```

Comment:
`state_partitions_query` and similar below
as well as `stateless_partitions_query` for ones below

Reply:
see #2268